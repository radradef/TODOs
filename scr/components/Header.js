import React from 'react';
import { Text, View, StyleSheet } from 'react-native';

const Header = () => {
    return (
        <View style={styles.header}>
            <Text style={styles.text}>
                TODOs:
            </Text>
        </View>
    );
}

const styles = StyleSheet.create({
    header: {
        height: 60,
        padding: 15,
        backgroundColor: 'darkslateblue'
    },
    text: {
        color: 'white',
        fontSize: 25,
        textAlign: 'center'
    }
});


export default Header;