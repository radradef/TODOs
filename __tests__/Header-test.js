/**
 * @format
 */

 import 'react-native';
 import React from 'react';
 import Header from '../scr/components/Header';
 import { expect } from '@jest/globals';
 import renderer from 'react-test-renderer';
 
 it('matches snapshot', () => {
   const tree = renderer.create(<Header />).toJSON();
   expect(tree).toMatchSnapshot();
 });
 