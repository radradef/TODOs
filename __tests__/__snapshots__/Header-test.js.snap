// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "darkslateblue",
      "height": 60,
      "padding": 15,
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 25,
        "textAlign": "center",
      }
    }
  >
    TODOs:
  </Text>
</View>
`;
